content/getting-started/building-a-snap-package.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Building a snap package
+description: How to build and release a snap package with codemagic.yaml
+weight: 11
+aliases:
+  - '../yaml/building-a-snap-package'
+---
+
+## Setting up snap packaging
+
+To set up snap packaging, create a `snapcraft.yaml` file with the necessary configurations according to [Snapcraft guide for Flutter](https://snapcraft.io/docs/flutter-applications) or follow the general [`snapcraft.yaml` guide](https://snapcraft.io/docs/creating-snapcraft-yaml).
+
+Optionally, run the `snapcraft snap` command locally to ensure that everything is set up.
+
+You should store the `snapcraft.yaml` file in the repository root. Another option is to store `snapcraft.yaml` in the `.snap` folder that is located in repository root.
+
+{{<notebox>}}
+See a sample application along with a `codemagic.yaml` configuration file that builds and releases a snap package in [GitHub](https://github.com/codemagic-ci-cd/flutter-snapcraft-example/).
+{{</notebox>}}
+
+## Building snap packages
+
+Include the `snapcraft snap` command in the `scripts` section of your `codemagic.yaml` file as in the example below. The output of this command is a `.snap` artifact. It can later be used for [publishing to the Snapcraft Snap Store](#publishing-snap-packages).
+
+```yaml
+workflows:
+  snap-build:
+    name: Snapcraft Build
+    instance_type: linux
+    environment:
+        SNAPCRAFT_BUILD_ENVIRONMENT: host
+    scripts:
+      - name: Create a snap
+        script: snapcraft snap --output flutter-codemagic-example.snap
+    artifacts:
+        - '**/*.snap'
+```
+
+Additionally, you may want to install the generated `.snap` package onto your machine. The package will not be code signed unless you publish it to Snapcraft. You would need to use the `--dangerous` flag to install the package without code signing:
```

Comment:
Is this for testing locally? If so can you add that here so people know why they would want to do this.